src/main/java/com/iexec/worker/sgx/SgxService.java

```diff
@@ -90,11 +128,18 @@ private boolean isSgxDevicePresent() {
 
         DockerRunResponse dockerRunResponse = dockerService.run(dockerRunRequest);
```

Comment:
Do we really need to map all devices?
https://github.com/iExecBlockchainComputing/iexec-worker/blob/23a4b347a64d9a23bb8549653e1890d24a5eba6d/src/main/java/com/iexec/worker/sgx/SgxService.java#L125
We would rather map `devices` list if possible.

Reply:
Sure, fixed in https://github.com/iExecBlockchainComputing/iexec-worker/pull/434/commits/044fd06268d03ff1ba6574be470d537e8567d061.